src/Raven.Client/Documents/Operations/TransactionsRecording/StartTransactionsRecordingOperation.cs

```diff
@@ -33,25 +33,29 @@ public StartTransactionsRecordingCommand(string filePath)
 
             public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, ServerNode node, out string url)
             {
-                url = $"{node.Url}/databases/{node.Database}/transactions/start-recording?file={_filePath}";
+                url = $"{node.Url}/databases/{node.Database}/admin/transactions/start-recording";
 
                 var request = new HttpRequestMessage
                 {
                     Method = HttpMethod.Post,
                     Content = new BlittableJsonContent(stream =>
                     {
-                        var jsonReaderObject = EntityToBlittable.ConvertEntityToBlittable(
-                                entity: new { file = _filePath },
-                                DocumentConventions.Default,
-                                ctx,
-                                new JsonSerializer(),
-                                documentInfo: null
+                        var jsonReaderObject = EntityToBlittable.ConvertCommandToBlittable(
+                            //Todo To think where it should be created
```

Comment:
What do you mean?

Reply:
Let's create an object here that will be passed from the caller. This will simplify versioning concerns in the future.

Reply:
So, do you mean to create the parameters object in the operation (StartTransactionsRecordingOperation) and pass it to the command (StartTransactionsRecordingCommand) ?

Reply:
done